Cargo.toml

```diff
@@ -18,4 +18,4 @@ keywords = ["regex", "match", "pattern", "streaming"]
 maintenance = { status = "experimental" }
 
 [dependencies]
-regex-automata = "0.1"
+regex-automata = { version = "0.4", default-features = false, features = ["syntax", "dfa-build", "dfa-search"] }
```

Comment:
We may want to add our own feature flag that enables the `regex-automata` unicode features, so that downstream dependents can enable unicode without needing to add an explicit dependency on `regex-automata`. Not a blocker for this PR, though.